pkg/version/compatibility.go

```diff
@@ -53,6 +53,18 @@ var CompatibilityMatrix = Compatibility{
 		"~1.3.0": {
 			Envoy: "~1.18.4",
 		},
+		"~1.4.0": {
+			Envoy: "~1.18.4",
+		},
+		// This includes all dev versions beyond 1.4.0 (master at the time
+		// of this change)
+		// It should be updated whenever a new Envoy version is introduced on
+		// master.
+		// Every time a release is made, a test in this package will begin to
+		// fail on master and must be updated.
+		"~1.4.0-1-anyprerelease": {
```

Comment:
How does it work? We won't have 1.4.0 tag on master

Reply:
The branch point is at `1.4.0-rc1` so we come up with a version like `1.4.0-300-g148bcfe` which is matched by this version.